### PR TITLE
Feature/registration for patients

### PR DESCRIPTION
**Pull request by Aleksa1998**

### CODE REVIEW

File: PatientWebApplication/PatientWebApplication/Controllers/PatientController.cs [hunk 1]
```diff
@@ -34,6 +37,20 @@ public IActionResult Create(PatientDto patientDto)
             return Ok();
             
 
+        }
+
+        [HttpPost("image")]
+        public IActionResult SaveImg([FromForm] FileModel file)
+        {
+            string pathToReturn = PatientService.ImageToSave(file);
+            if (pathToReturn == null)
```
Comment (trki293) [inline PatientWebApplication/PatientWebApplication/Controllers/PatientController.cs:46]: The variable 'pathToReturn' can be defined in an if statement.

File: PatientWebApplication/PatientWebApplication/Controllers/PatientController.cs [hunk 2]
```diff
@@ -25,7 +27,8 @@ public PatientUserController()
         [HttpPost]
         public IActionResult Create(PatientDto patientDto)
         {
-           
+
+
```
Comment (trki293) [inline PatientWebApplication/PatientWebApplication/Controllers/PatientController.cs:31]: Unnecessarily added blank lines need to be deleted.

**Review by trki293 — CHANGES_REQUESTED** [review]
Necessary changes in the indicated places.

**Review by trki293 — APPROVED** [review]
PR is approved, all Clean code principles are followed.

**Review by velicko-tam — APPROVED** [review]
PR is approved because of the good Clean code principles that were implemented in the solution.

**Pull request merged**